src/goto-symex/symex_main.cpp

```diff
@@ -331,6 +331,15 @@ void goto_symext::symex_step(reachability_treet &art)
         return;
       }
 
+      if (
+        id.as_string() == "c:@F@memcpy" ||
```

Comment:
With the other comment applied, this should be changed to just check on __ESBMC_memcpy (in the format that we expect).